modules/adgenerationBidAdapter.js

```diff
@@ -0,0 +1,194 @@
+import * as utils from 'src/utils';
+// import {config} from 'src/config';
+import {registerBidder} from 'src/adapters/bidderFactory';
+import { NATIVE } from 'src/mediaTypes';
+const ADG_BIDDER_CODE = 'adgeneration';
+
+export const spec = {
+  code: ADG_BIDDER_CODE,
+  aliases: ['adg'], // short code
+  supportedMediaTypes: [NATIVE],
```

Comment:
If you are supporting more than native, I.E. `banner`, you need to set it explicitly here.